lemur/common/utils.py

```diff
@@ -25,6 +25,7 @@
 from cryptography.hazmat.primitives.serialization import load_pem_private_key, Encoding, pkcs7
```

Comment:
This piece of code is quite CPU-intensive; let's discuss possible optimizations.